specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2023-03-01/costmanagement.benefits.json

```diff
@@ -656,7 +656,7 @@
         "tags": [
           "BenefitUtilizationSummariesAsync"
```

Comment:
How about removing `Async` word from the tags and from the descriptions in this operation to be consistent?